llama-rs/src/lib.rs

```diff
@@ -1065,11 +1077,15 @@ impl Model {
     }
 
     /// Evaluates the transformer.
+    ///
+    /// The provided `output_request` struct lets you specify which additional
+    /// data you are interested in fetching from the transformer. Setting a field to a `Some` value will fill the
```

Comment:
I think you might have run out of context tokens here. :(

Reply:
@setzer22 fyi

Reply:
Oops, missed that one, should be fixed now :+1: